source/FreeRTOS_IP_Utils.c

```diff
@@ -825,6 +829,10 @@ uint16_t usGenerateChecksum( uint16_t usSum,
     xTerm.u32 = 0UL;
 
     xSource.u8ptr = ipPOINTER_CAST( uint8_t *, pucNextData );
+
+    /* the conversion here does not cause a loss of data, as uintptr_t fits a
+     * pointer type on the system */
+    /* coverity[misra_c_2012_rule_11_1_violation] */
```

Comment:
MISRA 11.1 is in regards to function pointers. Shouldn't this be referencing 11.4?